stl/inc/vector

```diff
@@ -771,6 +771,24 @@ public:
 #endif // _ITERATOR_DEBUG_LEVEL != 0
     }
 
+public:
+    template <bool _Force_pass = false>
+    _NODISCARD static _CONSTEXPR20 vector _Create_uninitialized(size_type _Size, const allocator_type& _Al) {
+        // Create a vector of size _Size without value initialization.
+        // Currently used by basic_stacktrace.
+        static_assert(_Force_pass || is_trivially_constructible_v<_Ty>, "_Ty must be trivially constructible");
+
+        vector _Vec(_Al);
+        if (_Is_constant_evaluated()) {
+            _Vec.resize(_Size);
+        } else {
+            const pointer _Newvec = _Vec._Getal().allocate(_Size);
+            _Vec._Change_array(_Newvec, _Size, _Size);

```

Comment:
It seems that we can just befriend `basic_stacktrace` and then do these stuffs in the `_Internal_t` constructor.

Note that `_CONSTEXPR20` and `_Is_constant_evaluated()` shouldn't be needed since the new logic is only performed outside of constant evaluation.

Reply:
adopted in v3 branch